recipes/tokenizers/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "tokenizers" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 703101ffc1cce87e39a8fa9754126a5c29590b03817a73727e3268474dc716e6
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+  missing_dso_whitelist:
+    - /usr/lib/libresolv.9.dylib  # [osx]
+    - /usr/lib64/libgcc_s.so.1  # [linux]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+  host:
+    - python
+    - pip
+    - setuptools-rust
+    - setuptools
+  run:
+    - python
+    - libgcc-ng                                                  # [linux]
+
+test:
+  imports:
+    - tokenizers
+    - tokenizers.models
+    - tokenizers.decoders
+    - tokenizers.normalizers
+    - tokenizers.pre_tokenizers
+    - tokenizers.processors
+    - tokenizers.trainers
+    - tokenizers.implementations
+
+about:
+  home: https://github.com/huggingface/tokenizers
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "Fast and Customizable Tokenizers"
```

Comment:
Recommend this from the GitHub page: "Fast State-of-the-Art Tokenizers optimized for Research and Production"